Wordle.Api/Wordle.Api/Identity/Policies.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.AspNetCore.Authorization;
+
+namespace Wordle.Api.Identity
+{
+    public static class Policies
+    {
+
+        public const string RandomAdmin = "RandomAdmin";
+        public const string CanAddDeleteWords = "CanAddDeleteWords";
+
+        public static void RandomAdminPolicy(AuthorizationPolicyBuilder policy)
+        {
+            policy.RequireRole(Roles.Admin);
+            policy.RequireAssertion(context =>
+            {
+                var random = context.User.Claims.FirstOrDefault(c => c.Type == Claims.Random);
+
+                if (Double.TryParse(random?.Value, out double result))
+                {
+                    return result > 0.5;
+                }
+
+                return false;
+            });
+        }
+
+        public static void CanAddDeleteWordsPolicy(AuthorizationPolicyBuilder policy)
+        {
+
+            policy.RequireAssertion(context =>
+            {
+
+                var hasMasterClaim = context.User.Claims.FirstOrDefault(c => c.Type == Claims.MasterOfTheUniverse);
+                if (hasMasterClaim == null)
+                {
+                    return false;
+                }
+
+                // If hasMsterClaim == true then continue check
+                if (bool.TryParse(hasMasterClaim?.Value, out bool isMaster) && isMaster == false)
+                {
+                    return false;
+                }
+
+                var birthdateClaim = context.User.Claims.FirstOrDefault(c => c.Type == Claims.BirthDate);
+                if (birthdateClaim == null)
+                {
```

Comment:
again, great job! (sounding like a broken record) but I think the code looks clean and concise

Reply:
Thanks